### DISCUSSION TITLE
Add "Always On Top" as a pushpin icon next to the minimise, maximise and close icon in the program bar [Feature Request]

### DISCUSSION THREAD

**Discussion by mofosyne** [linuxmint/discussions#138] [Ideas]
I find that I would like to keep windows on top, especially terminal or widget style programs. Or text notes.
You already got "Always On Top" but it's a hidden in submenus.

I think it's worth exposing as a quick button with an icon that looks like a push pin. If you press it the icon should look like a pinned icon. 

![image](https://github.com/linuxmint/discussions/assets/827793/f4d35e82-3275-47c5-8e79-77ae3a76a396)

![image](https://github.com/linuxmint/discussions/assets/827793/f54c9b9a-8d2b-465d-b09f-df6f77f4fdf8)

**Comment by Secret-chest** [comment]
Having it always visible would be overkill. And I don't think GTK+ supports more buttons.
